css/style.css

```diff
@@ -0,0 +1,307 @@
+:root {
+    /* colors */
+    --black: #000000;
+    --white: #ffffff;
+    --wine: #823468;
+    --pink-light: #F2B8CC;
+    --cream-light: #F6F4E9;
+    
+    /*  fw-fonts-weight */
+    --fw-400: 400;
+    --fw-500: 600;
+    --fw-700: 700;
+    --fw-900: 900;
+   
+    /* fs-font-size */
+    --fs-36px: 2.25rem;
+    --fs-24px: 1.5rem;
+    --fs-20px: 1.25rem;
+    --fs-12px: 0.75rem;
```

Comment:
love the use of variables! 
How can you make the names of these font size & weight variables more meaningful? For example, if you use one of these font size variables to set a header font-size, how do you know which one to use? You could give the variables names that guide you when to use each size